### PR TITLE
Real quick edits for in-person

### PR DESCRIPTION
**Pull request by plaindocs**
Could use venue specs also. 

I'd like to merge this asap so I can link it in the next speaker mail. 

----
:books: Documentation preview :books:: https://writethedocs-www--1914.org.readthedocs.build/

### CODE REVIEW

File: docs/include/conf/portland/speaking-tips.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+-  Your talk will be 30 minutes long.
+   Your talk should cover the entire slot, or close to it. 
+-  There will be a moderated live Q&A, around 10 minutes long, immediately after your talk.
+-  The more you practice your talk, the more comfortable you'll be when giving it.
+   In addition to practicing by yourself, we strongly recommend you
+   run through it in front of at least one other human.
+-  If you want to get some of that sweet internet buzz for your talk,
+   we use the #writethedocs hashtag throughout the event.
+
```
Comment (plaindocs) [inline docs/include/conf/portland/speaking-tips.rst:18]: Venue specs
-------------------

The projector is 1920 x 1200 resolution at 60Hz via HDMI, DisplayPort or VGA

Comment (plaindocs) [inline docs/include/conf/portland/speaking-tips.rst:18]:
```suggestion
Venue specs
------------------

The projector is 1920 x 1200 resolution at 60Hz via HDMI, DisplayPort or VGA
```

File: docs/include/conf/portland/speaking-tips.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+-  Your talk will be 30 minutes long.
+   Your talk should cover the entire slot, or close to it. 
+-  There will be a moderated live Q&A, around 10 minutes long, immediately after your talk.
+-  The more you practice your talk, the more comfortable you'll be when giving it.
+   In addition to practicing by yourself, we strongly recommend you
+   run through it in front of at least one other human.
+-  If you want to get some of that sweet internet buzz for your talk,
+   we use the #writethedocs hashtag throughout the event.
+Venue specs
```
Comment (plaindocs) [inline docs/include/conf/portland/speaking-tips.rst:19]:
```suggestion

Venue specs
```

File: docs/include/conf/portland/speaking-tips.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+language that's presumptive about or exclusive towards the variety of folks who will be in the audience.
+
+Good resources on this include:
+
+- `The 18F Guide to Inclusive Language <https://content-guide.18f.gov/inclusive-language/>`__
+- `The Responsible Communication Style Guide <https://rcstyleguide.com/>`__
+- `GDS design principles <https://www.gov.uk/guidance/government-design-principles#this-is-for-everyone>`__
+
+Also, we know there's a ton of nuance and complexity here – 
+just do your best to be aware of and sensitive about your language choices!
```
Comment (mxsasha) [inline docs/include/conf/portland/speaking-tips.rst:51]:
```suggestion
just do your best to be aware of and sensitive about your language choices!
If you have any doubts about any of your language, feel free to ask us in advance.
```

**Review by mxsasha — APPROVED** [review]

**Pull request merged**